frontend/src/authentication/ProfileSetUp.jsx

```diff
@@ -0,0 +1,192 @@
+import { useState } from "react";
```

Comment:
1. That PR was for the profile page/profile edit. This one is part of the registration flow where the user inputs their details for the first time.
2. I mentioned this in the workspace PR group. When I made the commit, it keep including these code, even though it was not staged. The actual code for this PR is the code listed in the PR description (the backend code). The ProfileSetUp component is in PR #17. The frontend code here is not ready. I was hoping I can resolve this as a merge conflict when I merge the frontend PR into this PR, before merging to main.

Reply:
PR #17 is the child PR of this one right? You should be able to remove these changes from this PR and have them in the frontend PR, since the other frontend changes are not included in this PR. And you don't need to merge the frontend PR into this PR, before merging to main. You can merge this PR into main, and then merge the frontend PR

Reply:
I reset the initial commit, removed the unrelated file and I have now created a new commit: https://github.com/izuchukwumba/peered/pull/16/commits/0f8e5bdf8b30bf2485d88352177bed646319fb35